Cannot unzip on Windows if user directory contains whitespace

### Preflight Checklist

* [x] I have read the [contribution documentation](https://github.com/electron/electron-packager/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [code of conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project follows, as appropriate.
* [x] I have searched the issue tracker for a bug that matches the one I want to file, without success.

### Issue Details

* **Electron Packager Version:**
  * 14.2.1
* **Electron Version:**
  * v8.0.0
* **Operating System:**
  * Windows 10 (1903)
* **Last Known Working Electron Packager version:**:
  * unknown

### Expected Behavior
electron-packager should work

### Actual Behavior
electron-packager terminates with

```
[Error: ENOENT: no such file or directory, rename 'C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64\VIPFY-win32-x64\electron.exe' -> 'C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64\VIPFY-win32-x64\VIPFY.exe'] {      
  errno: -4058,
  code: 'ENOENT',
  syscall: 'rename',
  path: 'C:\\Users\\JANNIS~1\\AppData\\Local\\Temp\\electron-packager\\win32-x64\\VIPFY-win32-x64\\electron.exe',
  dest: 'C:\\Users\\JANNIS~1\\AppData\\Local\\Temp\\electron-packager\\win32-x64\\VIPFY-win32-x64\\VIPFY.exe'
}
```

### To Reproduce
Use a user directory with whitespace, or alternatively set APPDATALOCAL environment variable to a directory with whitespace

For example in Powershell:
```
$env:LOCALAPPDATA = "C:\test\folder with whitespace\"
# ensure this folder exists
electron-packager . #or any electron-packager call that normally works
```

### Additional Information
Relevant console output:

```
electron-packager Creating C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64-template +16ms
  electron-packager Extracting C:\Users\Jannis Froese\AppData\Local\electron\Cache\httpsgithub.comelectronelectronreleasesdownloadv8.0.0-beta.4electron-v8.0.0-beta.4-win32-x64.zip\electron-v8.0.0-beta.4-win32-x64.zip to C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64-template +2ms
  electron-packager Ignored paths based on the out param: [ 'C:\\Users\\Jannis Froese\\Source\\vipfy-desktop\\out' ] +0ms

Error: ENOENT: no such file or directory, rename 'C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64\VIPFY-win32-x64\electron.exe' -> 'C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64\VIPFY-win32-x64\VIPFY.exe'] {      
  errno: -4058,
  code: 'ENOENT',
  syscall: 'rename',
  path: 'C:\\Users\\JANNIS~1\\AppData\\Local\\Temp\\electron-packager\\win32-x64\\VIPFY-win32-x64\\electron.exe',
  dest: 'C:\\Users\\JANNIS~1\\AppData\\Local\\Temp\\electron-packager\\win32-x64\\VIPFY-win32-x64\\VIPFY.exe'
}
```

Manually running cross-zip-cli with the exact paths as shown in the debug output extracts the zip file correctly

Sure
```
PS C:\VIPFY\vipfy-desktop> $env:DEBUG = "electron-packager"
PS C:\VIPFY\vipfy-desktop> npm run package

> vipfy-desktop@0.11.18 package C:\VIPFY\vipfy-desktop
> electron-forge package

√ Checking your system
√ Compiling Main Process Code
√ Compiling Renderer Template
  electron-packager Electron Packager 14.2.1
  electron-packager Node v12.7.0
  electron-packager Host Operating system: win32 10.0.18362 (x64) +0ms
  electron-packager Packager Options: {"asar":true,"overwrite":true,"icon":"iconTransparent","appCopyright":"©2020 VIPFY GmbH","osxSign":{"platform":"darwin","type":"distribution","identity":"Developer ID Application: VIPFY GmbH (RD6VS27844)"},"dir":"C:\\VIPFY\\vipfy-desktop","arch":"x64","platform":"win32","afterCopy":[null],"afterExtract":[null],"afterPrune":[null],"out":"C:\\VIPFY\\vipfy-desktop\\out","electronVersion":"8.0.0-beta.7","quiet":true} +1ms
  electron-packager Target Platforms: win32 +1ms
  electron-packager Target Architectures: x64 +0ms
  electron-packager Inferring application name from productName in C:\VIPFY\vipfy-desktop\package.json +0ms
  electron-packager Inferring appVersion from version in C:\VIPFY\vipfy-desktop\package.json +0ms
  electron-packager Inferring win32metadata.CompanyName from author in C:\VIPFY\vipfy-desktop\package.json +0ms
  electron-packager Application name: VIPFY +11ms
  electron-packager Target Electron version: 8.0.0-beta.7 +0ms
  electron-packager Downloading Electron with options {"platform":"win32","arch":"x64","version":"8.0.0-beta.7","artifactName":"electron"} +0ms
  electron-packager Removing C:\VIPFY\vipfy-desktop\out\VIPFY-win32-x64 due to setting overwrite: true +20ms
| Preparing to Package Application for arch: x64  electron-packager Creating C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64-template +47ms
  electron-packager Extracting C:\Users\Jannis Froese\AppData\Local\electron\Cache\httpsgithub.comelectronelectronreleasesdownloadv8.0.0-beta.7electron-v8.0.0-beta.7-win32-x64.zip\electron-v8.0.0-beta.7-win32-x64.zip to C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64-template +1ms
\ Preparing to Package Application for arch: x64  electron-packager Initializing app in C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64\VIPFY-win32-x64 from C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64-template template +0ms
  electron-packager Ignored paths based on the out param: [ 'C:\\VIPFY\\vipfy-desktop\\out' ] +0ms
√ Preparing to Package Application for arch: x64
√ Preparing native dependencies
- Packaging Application  electron-packager Running asar with the options {} +404ms
/ Packaging Application  electron-packager Renaming electron.exe to VIPFY.exe in C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64\VIPFY-win32-x64 +483ms

An unhandled rejection has occurred inside Forge:
[Error: ENOENT: no such file or directory, rename 'C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64\VIPFY-win32-x64\electron.exe' -> 'C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64\VIPFY-win32-x64\VIPFY.exe'] {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'rename',
  path: 'C:\\Users\\JANNIS~1\\AppData\\Local\\Temp\\electron-packager\\win32-x64\\VIPFY-win32-x64\\electron.exe',
  dest: 'C:\\Users\\JANNIS~1\\AppData\\Local\\Temp\\electron-packager\\win32-x64\\VIPFY-win32-x64\\VIPFY.exe'
}

Electron Forge was terminated. Location:
{}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vipfy-desktop@0.11.18 package: `electron-forge package`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the vipfy-desktop@0.11.18 package script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Jannis~1\AppData\Roaming\npm-cache\_logs\2020-02-11T15_54_45_944Z-debug.log
PS C:\VIPFY\vipfy-desktop>
```

```
PS C:\VIPFY\vipfy-desktop> $env:DEBUG = "electron-packager,electron-forge:*"
PS C:\VIPFY\vipfy-desktop> npm run package

> vipfy-desktop@0.11.18 package C:\VIPFY\vipfy-desktop
> electron-forge package

WARNING: DEBUG environment variable detected.  Progress indicators will be sent over electron-forge:lifecycle
  electron-forge:async-ora Process Started: Checking your system +0ms
  electron-forge:check-system checking system, create ~/.skip-forge-system-check to stop doing this +0ms
  electron-forge:async-ora Process Succeeded: Checking your system -- after 222ms +223ms
WARNING: DEBUG environment variable detected.  Progress indicators will be sent over electron-forge:lifecycle
  electron-forge:async-ora Process Started: Preparing to Package Application for arch: x64 +0ms
  electron-forge:project-resolver searching for project in: C:\VIPFY\vipfy-desktop +0ms
  electron-forge:project-resolver electron-forge compatible package.json found in C:\VIPFY\vipfy-desktop\package.json +2ms
  electron-forge:require-search searching [
  '@electron-forge/plugin-webpack',
  'C:\\VIPFY\\vipfy-desktop\\@electron-forge\\plugin-webpack',
  'C:\\VIPFY\\vipfy-desktop\\node_modules\\@electron-forge\\plugin-webpack'
] relative to C:\VIPFY\vipfy-desktop +0ms
  electron-forge:require-search testing @electron-forge/plugin-webpack +2ms
  electron-forge:plugin:webpack hooking process events +0ms
  electron-forge:async-ora Process Started: Compiling Main Process Code +1s
  electron-forge:async-ora Process Succeeded: Compiling Main Process Code -- after 3.3s +3s
  electron-forge:async-ora Process Started: Compiling Renderer Template +0ms
  electron-forge:async-ora Process Succeeded: Compiling Renderer Template -- after 16.7s +17s
  electron-forge:packager packaging with options {
  asar: true,
  overwrite: true,
  icon: 'iconTransparent',
  appCopyright: '©2020 VIPFY GmbH',
  osxSign: {
    platform: 'darwin',
    type: 'distribution',
    identity: 'Developer ID Application: VIPFY GmbH (RD6VS27844)'
  },
  ignore: [Function],
  dir: 'C:\\VIPFY\\vipfy-desktop',
  arch: 'x64',
  platform: 'win32',
  afterCopy: [ [AsyncFunction] ],
  afterExtract: [ [AsyncFunction] ],
  afterPrune: [ [AsyncFunction] ],
  out: 'C:\\VIPFY\\vipfy-desktop\\out',
  electronVersion: '8.0.0-beta.7',
  quiet: true
} +0ms
  electron-packager Electron Packager 14.2.1
  electron-packager Node v12.7.0
  electron-packager Host Operating system: win32 10.0.18362 (x64) +0ms
  electron-packager Packager Options: {"asar":true,"overwrite":true,"icon":"iconTransparent","appCopyright":"©2020 VIPFY GmbH","osxSign":{"platform":"darwin","type":"distribution","identity":"Developer ID Application: VIPFY GmbH (RD6VS27844)"},"dir":"C:\\VIPFY\\vipfy-desktop","arch":"x64","platform":"win32","afterCopy":[null],"afterExtract":[null],"afterPrune":[null],"out":"C:\\VIPFY\\vipfy-desktop\\out","electronVersion":"8.0.0-beta.7","quiet":true} +2ms
  electron-packager Target Platforms: win32 +2ms
  electron-packager Target Architectures: x64 +1ms
  electron-packager Inferring application name from productName in C:\VIPFY\vipfy-desktop\package.json +0ms
  electron-packager Inferring appVersion from version in C:\VIPFY\vipfy-desktop\package.json +1ms
  electron-packager Inferring win32metadata.CompanyName from author in C:\VIPFY\vipfy-desktop\package.json +0ms
  electron-packager Application name: VIPFY +24ms
  electron-packager Target Electron version: 8.0.0-beta.7 +0ms
  electron-packager Downloading Electron with options {"platform":"win32","arch":"x64","version":"8.0.0-beta.7","artifactName":"electron"} +0ms
  electron-packager Creating C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64-template +15ms
  electron-packager Extracting C:\Users\Jannis Froese\AppData\Local\electron\Cache\httpsgithub.comelectronelectronreleasesdownloadv8.0.0-beta.7electron-v8.0.0-beta.7-win32-x64.zip\electron-v8.0.0-beta.7-win32-x64.zip to C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64-template +16ms
  electron-packager Initializing app in C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64\VIPFY-win32-x64 from C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64-template template +0ms
  electron-packager Ignored paths based on the out param: [ 'C:\\VIPFY\\vipfy-desktop\\out' ] +0ms
  electron-forge:async-ora Process Succeeded: Preparing to Package Application for arch: x64 -- after 22.5s +1s
  electron-forge:async-ora Process Started: Preparing native dependencies +3ms
  electron-forge:async-ora Process Succeeded: Preparing native dependencies -- after 16ms +16ms
  electron-forge:async-ora Process Started: Packaging Application +9ms
  electron-packager Running asar with the options {} +727ms
  electron-packager Renaming electron.exe to VIPFY.exe in C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64\VIPFY-win32-x64 +508ms

An unhandled rejection has occurred inside Forge:
[Error: ENOENT: no such file or directory, rename 'C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64\VIPFY-win32-x64\electron.exe' -> 'C:\Users\JANNIS~1\AppData\Local\Temp\electron-packager\win32-x64\VIPFY-win32-x64\VIPFY.exe'] {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'rename',
  path: 'C:\\Users\\JANNIS~1\\AppData\\Local\\Temp\\electron-packager\\win32-x64\\VIPFY-win32-x64\\electron.exe',
  dest: 'C:\\Users\\JANNIS~1\\AppData\\Local\\Temp\\electron-packager\\win32-x64\\VIPFY-win32-x64\\VIPFY.exe'
}

Electron Forge was terminated. Location:
{}
  electron-forge:plugin:webpack handling process exit with: { cleanup: true } +22s
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vipfy-desktop@0.11.18 package: `electron-forge package`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the vipfy-desktop@0.11.18 package script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Jannis~1\AppData\Roaming\npm-cache\_logs\2020-02-11T16_59_31_784Z-debug.log
PS C:\VIPFY\vipfy-desktop>
```

--- a/README.md
+++ b/README.md
@@ -52,7 +52,7 @@ It generates executables/bundles for the following **target** platforms:
 
 ## Installation
 
-This module requires Node.js 10.0 or higher to run. On macOS/Linux, the `unzip` program is required. On Windows, both [.NET Framework 4.5 or higher and Powershell 3 or higher are required](https://github.com/feross/cross-zip#windows-users).
+This module requires Node.js 10.0 or higher to run.
 
 ```sh
 # For use in npm scripts (recommended)

--- a/package.json
+++ b/package.json
@@ -29,10 +29,10 @@
   "dependencies": {
     "@electron/get": "^1.6.0",
     "asar": "^3.0.0",
-    "cross-zip": "^3.0.0",
     "debug": "^4.0.1",
     "electron-notarize": "^0.2.0",
     "electron-osx-sign": "^0.4.11",
+    "extract-zip": "^2.0.0",
     "fs-extra": "^9.0.0",
     "galactus": "^0.2.1",
     "get-package-info": "^1.0.0",

--- a/src/unzip.js
+++ b/src/unzip.js
@@ -1,31 +1,7 @@
 'use strict'
 
-const os = require('os')
-const { promisify } = require('util')
-const { warning } = require('./common')
-const zip = require('cross-zip')
-
-const unzip = promisify(zip.unzip)
-
-/**
- * Detects Windows 7 via release number.
- *
- * This also detects Windows Server 2008 R2, but since we're using it to determine whether to check * for Powershell/.NET Framework, it's fine.
- */
-function probablyWindows7 () {
-  if (process.platform === 'win32') {
-    const [majorVersion, minorVersion] = os.release().split('.').map(Number)
-    return majorVersion === 6 && minorVersion === 1
-  }
-
-  return false
-}
+const extractZip = require('extract-zip')
 
 module.exports = async function extractElectronZip (zipPath, targetDir) {
-  if (probablyWindows7()) {
-    /* istanbul ignore next */
-    warning('Make sure that .NET Framework 4.5 or later and Powershell 3 or later are installed, otherwise extracting the Electron zip file will hang.')
-  }
-
-  return unzip(zipPath, targetDir)
+  await extractZip(zipPath, { dir: targetDir })
 }

--- a/test/index.js
+++ b/test/index.js
@@ -10,6 +10,7 @@ require('./infer')
 require('./hooks')
 require('./prune')
 require('./targets')
+require('./unzip')
 require('./win32')
 
 if (process.platform !== 'win32') {

--- a/test/unzip.js
+++ b/test/unzip.js
@@ -1,0 +1,24 @@
+'use strict'
+
+const download = require('../src/download')
+const unzip = require('../src/unzip')
+const fs = require('fs-extra')
+const config = require('./config.json')
+const path = require('path')
+const os = require('os')
+const test = require('ava')
+
+for (const downloadOpts of download.createDownloadCombos({ electronVersion: config.version }, ['darwin', 'mas'], ['x64'])) {
+  test.serial(`unzip preserves symbolic links (${downloadOpts.platform})`, t => { return unzipPreserveSymbolicLinks(t, downloadOpts) })
+}
+
+async function unzipPreserveSymbolicLinks (t, downloadOpts) {
+  const zipPath = await download.downloadElectronZip(downloadOpts)
+  const tempPath = await fs.mkdtemp(path.join(os.tmpdir(), 'symlinktest-'))
+
+  await unzip(zipPath, tempPath)
+
+  const testSymlinkPath = path.join(tempPath, 'Electron.app/Contents/Frameworks/Electron Framework.framework/Libraries')
+  const stat = await fs.lstat(testSymlinkPath)
+  t.true(stat.isSymbolicLink(), 'extract symoblic links')
+}

--- a/test/unzip.js
+++ b/test/unzip.js
@@ -1,24 +1,17 @@
 'use strict'
 
-const download = require('../src/download')
-const unzip = require('../src/unzip')
-const fs = require('fs-extra')
+const { assertSymlink } = require('./_util')
 const config = require('./config.json')
+const download = require('../src/download')
 const path = require('path')
-const os = require('os')
 const test = require('ava')
+const unzip = require('../src/unzip')
 
-for (const downloadOpts of download.createDownloadCombos({ electronVersion: config.version }, ['darwin', 'mas'], ['x64'])) {
-  test.serial(`unzip preserves symbolic links (${downloadOpts.platform})`, t => { return unzipPreserveSymbolicLinks(t, downloadOpts) })
-}
-
-async function unzipPreserveSymbolicLinks (t, downloadOpts) {
+test('unzip preserves symbolic links', async t => {
+  const downloadOpts = download.createDownloadOpts({ electronVersion: config.version }, 'darwin', 'x64')
   const zipPath = await download.downloadElectronZip(downloadOpts)
-  const tempPath = await fs.mkdtemp(path.join(os.tmpdir(), 'symlinktest-'))
 
-  await unzip(zipPath, tempPath)
+  await unzip(zipPath, t.context.tempDir)
 
-  const testSymlinkPath = path.join(tempPath, 'Electron.app/Contents/Frameworks/Electron Framework.framework/Libraries')
-  const stat = await fs.lstat(testSymlinkPath)
-  t.true(stat.isSymbolicLink(), 'extract symoblic links')
-}
+  await assertSymlink(t, path.join(t.context.tempDir, 'Electron.app/Contents/Frameworks/Electron Framework.framework/Libraries'), 'symbolic link extracted correctly')
+})

